Sources/ProjectDescription/Dependencies/SwiftPackageManagerDependencies.swift

```diff
@@ -29,25 +29,31 @@ public struct SwiftPackageManagerDependencies: Codable, Equatable {
     /// Custom project configurations to be used for projects generated from SwiftPackageManager.
     public let projectOptions: [String: ProjectDescription.Project.Options]
 
+    /// Allows including test targets into generated project for a swift package. This is useful for testing local swift packages in same repository
+    public let testableTargetsFromPackages: [String]
```

Comment:
This can be a `Set<String>`

Reply:
From the name, it's not clear whether you should put packages or targets name here, what about calling this `testsFromPackages`?